src/column/AColumn.ts

```diff
@@ -45,12 +49,23 @@ abstract class AColumn<T, DATATYPE extends IDataType> extends EventHandler {
   selectedUnaggVis: IVisPluginDesc;
   matrixFilters: AnyFilter[];//For the header in matrix
 
+  private _width:number = this.maxWidth;
```

Comment:
I used the maxWidth, because it was also set as attribute in the `build()` function.

https://github.com/Caleydo/taggle/blob/80f2466f727a5d109769ea1c15080e6fefe1d7c7/src/column/AColumn.ts#L130

But this value will anyway be overridden in `ColumnManager.relayout()`.